packages/design-system/src/styles/theme/oc-label.scss

```diff
@@ -0,0 +1,4 @@
+.oc-label:not(:empty) {
+  margin-bottom: 0.4em;
```

Comment:
could you switch to `rem` here? would like to avoid `em` if possible since it has different results depending on the container element.